src/biome/data/sources/readers.py

```diff
@@ -86,13 +97,18 @@ def from_parquet(path: Union[str, List[str]], **params) -> DataFrame:
     df
         A `dask.DataFrame`
     """
-    ddf = dd.read_parquet(path, **params, engine="pyarrow")
-    ddf["path"] = path
+    path_list = _get_file_paths(path)
```

Comment:
for `read_parquet` too https://docs.dask.org/en/latest/dataframe-api.html#dask.dataframe.read_parquet